models/edu/stanford/nlp/models/sutime/english.sutime.txt

```diff
@@ -888,37 +898,52 @@
 	           ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
     result: $0[-1].temporal.value
   }
-
   { name: "temporal-composite-7a",
     pattern: ( /every/ ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
-	result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
+	  result: MakePeriodicTemporalSet($1[0].temporal, "every", 1 )
   }
-
   { name: "temporal-composite-7b",
-#    pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
+#   pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal & !{ temporal::IS_TIMEX_SET } ] ) ),
     pattern: ( ( $FREQ_MOD ) ( [ $hasTemporal ] ) ),
-	result: MakePeriodicTemporalSet($2[0].temporal, GetTag($1[0], "PTS.quant"), GetTag($1[0], "PTS.multiple") )
+	  result: MakePeriodicTemporalSet($2[0].temporal, GetTag($1[0], "PTS.quant"), GetTag($1[0], "PTS.multiple") )
+  }
+  { name: "temporal-composite-8a",
```

Comment:
There's a lot going on here. Can you add a comment above each rule describing what case it's trying to solve?

Reply:
I've added a comment (and some examples) to each rule that I've added

Reply:
I've added comments and examples to each of the added rules